src/main/java/graphql/execution/ExecutionStepInfo.java

```diff
@@ -194,6 +200,15 @@ public String toString() {
         private Builder() {
         }
```

Comment:
What about a newXXX version of this

```
    public static ExecutionStepInfo.Builder newExecutionStepInfo(ExecutionStepInfo existing) {
        return new Builder(existing);
    }
```

just for completeness??  We do this in other places I think